src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+public enum Operation {
```

Comment:
It has to be a subclass of FruitTransaction

Reply:
When I solving this task, I was guided by the readme file. It contains recommendations, but it also says that you can solve this problem at your own discretion. I used the OpenCSV library as it significantly reduces the number of steps and lines of code. If these recommendations need to be followed, I will change my code. But I would like to hear opinion of the mentor too.